### PR TITLE
added mkr 485 shield guide

### PR DESCRIPTION
**Pull request by jhansson-ard**
## What This PR Changes
Added missing content for hardware

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/01.mkr/02.shields/mkr-485-shield/tutorials/getting-started/content.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+title: 'Getting Started with the MKR 485 Shield'
+description: 'Get started easily with the MKR 485 Shield.'
+tags:
+    - RS485
+author: 'Arduino'
+---
+
+The [MKR 485 shield](https://store.arduino.cc/arduino-mkr-485-shield) allows a MKR board to connect to other 485 devices.
+
+This board may be used with the [Arduino RS485](/en/Reference/ArduinoRS485) and the [Arduino Modbus](/en/ArduinoModbus/ArduinoModbus) libraries, available from the Library Manager.
```
Comment (karlsoderby) [inline content/hardware/01.mkr/02.shields/mkr-485-shield/tutorials/getting-started/content.md:11]: Old links that doesn't work anymore

File: content/hardware/01.mkr/02.shields/mkr-485-shield/tutorials/getting-started/content.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+![The MKR 485 Shield](assets/MKR485_featured.jpg)
+
+### Tutorials
+
+Now that you have a basic knowledge of the shield, you may find inspiration in our [Project Hub](https://projecthub.arduino.cc/) tutorial platform.
+
+<iframe frameborder='0' height='410' scrolling='no' src='https://create.arduino.cc/projecthub/123325/monitor-your-energy-bill-via-modbus-mkr-wifi-1010-and-rs485-814e5e/embed?use_route=project' width='354' style='margin-top:30px'></iframe>
+
+This shield may be used with the libraries already available for 485 communication.
```
Comment (karlsoderby) [inline content/hardware/01.mkr/02.shields/mkr-485-shield/tutorials/getting-started/content.md:35]:
```suggestion
## Use Cases

With the MKR 485 Shield you can interface with devices supporting the 485 standard. In the tutorial in the link below, you can learn how to interface with an **energy meter** using the Modbus protocol, and send the data to the Arduino Cloud.

- [Modbus Energy Meter and Arduino Cloud](https://docs.arduino.cc/arduino-cloud/application-notes/modbus-energy-meter/)
```

File: content/hardware/01.mkr/02.shields/mkr-485-shield/tutorials/getting-started/content.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+tags:
+    - RS485
+author: 'Arduino'
+---
+
+The [MKR 485 shield](https://store.arduino.cc/arduino-mkr-485-shield) allows a MKR board to connect to other 485 devices.
+
+This board may be used with the [Arduino RS485](/en/Reference/ArduinoRS485) and the [Arduino Modbus](/en/ArduinoModbus/ArduinoModbus) libraries, available from the Library Manager.
+
+### Usage Notes
```
Comment (karlsoderby) [inline content/hardware/01.mkr/02.shields/mkr-485-shield/tutorials/getting-started/content.md:13]:
```suggestion
## Usage Notes

```

File: content/hardware/01.mkr/02.shields/mkr-485-shield/tutorials/getting-started/content.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+The Arduino MKR 485 Shield allows the Arduino MKR family of boards to communicate with industrial automation systems or to extend the serial wired communication over much longer range. This shield supports half and full duplex with or without biasing and termination, master slave configuration.
+
+Most of these configurations are physical and the 3 ways dip switch allows you to set up the connection properly.
+
+| Switch No. | ON                   | OFf            |
```
Comment (karlsoderby) [inline content/hardware/01.mkr/02.shields/mkr-485-shield/tutorials/getting-started/content.md:19]:
```suggestion
| Switch No. | ON                   | OFF            |

```

**Review by karlsoderby — CHANGES_REQUESTED** [review]
Couple of suggestions and a couple of broken links found!

File: content/hardware/01.mkr/02.shields/mkr-485-shield/tutorials/getting-started/content.md (hunk 1 above)
Comment (jhansson-ard) [reply content/hardware/01.mkr/02.shields/mkr-485-shield/tutorials/getting-started/content.md:11]:
```suggestion
This board may be used with the [Arduino RS485](https://www.arduino.cc/reference/en/libraries/arduinors485/) and the [Arduino Modbus](https://www.arduino.cc/reference/en/libraries/arduinomodbus/) libraries, available from the Library Manager.

```

Comment (jhansson-ard) [reply content/hardware/01.mkr/02.shields/mkr-485-shield/tutorials/getting-started/content.md:11]: Are these correct @karlsoderby ?

**Review by karlsoderby — APPROVED** [review]
LGTM!

**Pull request merged**